spyder/plugins/editor/widgets/editor.py

```diff
@@ -1260,7 +1259,7 @@ def set_focus_to_editor(self, state):
 
     def set_run_cell_copy(self, state):
         """If `state` is ``True``, code cells will be copied to the console."""
-        self.run_cell_copy = state
+        CONF.set('editor', 'run_cell_copy', state)

```

Comment:
I don't like this change. In general we prefer widgets to not handle `CONF` changes because that's the responsibility of plugins.

Could you revert this and the associated changes you did here?